<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripleStoreDialog</class>
 <widget class="QDialog" name="TripleStoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="tripleStoreChooserLabel">
     <property name="text">
      <string>Choose Triple Store:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QComboBox" name="tripleStoreChooser"/>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="deleteTripleStore">
     <property name="text">
      <string>Delete Triple Store</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="tripleStoreNameLabel">
     <property name="text">
      <string>Triple Store Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="tripleStoreNameEdit">
     <property name="text">
      <string>My cool triplestore!</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="newTripleStore">
     <property name="text">
      <string>New Triple Store</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="tripleStoreLabel">
     <property name="text">
      <string>Triple Store URL:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="tripleStoreEdit">
     <property name="text">
      <string>https://query.wikidata.org/sparql</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="testConnectButton">
     <property name="text">
      <string>Test Connection</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="queryVarEdit">
     <property name="text">
      <string>geo</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="queryVarItemEdit">
     <property name="text">
      <string>item</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6" rowspan="2">
    <widget class="QPushButton" name="detectConfiguration">
     <property name="text">
      <string>Detect Configuration</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="epsgLabel">
     <property name="text">
      <string>EPSG Code:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="epsgEdit">
     <property name="text">
      <string>4326</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="activeTripleStoreLabel">
     <property name="text">
      <string>Active:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QCheckBox" name="activeCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="tripleStorePrefixNameLabel">
     <property name="text">
      <string>Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="tripleStorePrefixNameEdit">
     <property name="text">
      <string>wd</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="4">
    <widget class="QLineEdit" name="tripleStorePrefixEdit">
     <property name="text">
      <string>http://www.wikidata.org/entity/</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="addPrefixButton">
     <property name="text">
      <string>Add Prefix</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="removePrefixButton">
     <property name="text">
      <string>Remove Selected Prefix</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="prefixListLabel">
     <property name="text">
      <string>Prefixes:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="5">
    <widget class="QLabel" name="exampleQueryLabel">
     <property name="text">
      <string>Example Query (optional):</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QListWidget" name="prefixList"/>
   </item>
   <item row="10" column="2" colspan="5">
    <widget class="QPlainTextEdit" name="exampleQuery"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QPushButton" name="resetConfiguration">
     <property name="text">
      <string>Restore Default Triple Store Configuration</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="4">
    <widget class="QPushButton" name="tripleStoreApplyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="11" column="6">
    <widget class="QPushButton" name="tripleStoreCloseButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="queryVarItemLabel">
     <property name="text">
      <string>Item Variable:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="queryVarLabel">
     <property name="text">
      <string>Geometry Variable:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
